service/src/test/java/bio/terra/workspace/pact/TpsApiTest.java

```diff
@@ -182,26 +182,25 @@ public void deletingAnExistingPolicy() throws Exception {
   }
 
   @Pact(consumer = "wsm", provider = "tps")
-  public RequestResponsePact getPaoWithAnExistingPolicy(PactDslWithProvider builder) {
+  public V4Pact getPaoWithAnExistingPolicy(PactBuilder builder) {
     var policyResponseShape =
         new PactDslJsonBody()
-            .valueFromProviderState("objectId", existingPolicyProviderStateValue, existingPolicyId)
+            .stringValue("objectId", existingPolicyId.toString())
```

Comment:
Since the consumer is providing the `policyId` in the `getPao` of the test that uses this pact, it doesn't actually make sense that the provider would provide a value for this.  Instead, we accomplish communicating the expected policy ID using the parameterized `given` on line 196.

I made this change throughout.

Reply:
Fixed by e419a3404c191f2765a7411e50cb2a1861a05763